code.js

```diff
@@ -1,3 +1,7 @@
+//code.js
+//Maxie Machado 
+// Dijkstra's Algorithm 
+
 function dijkstra(graph, sourceNode) 
```

Comment:
When I run the code, It has an error. This is related to "If (!vis.has(node) && dist[node] < minDist)", javascript is case-sensitive, I think it should be if instead of If.